package.json

```diff
@@ -13,7 +13,7 @@
     "build:dll": "cross-env NODE_ENV=development TS_NODE_TRANSPILE_ONLY=true webpack --config ./webpack_configs/webpack.config.renderer.dev.dll.ts",
     "build:main": "cross-env NODE_ENV=production TS_NODE_TRANSPILE_ONLY=true webpack --config ./webpack_configs/webpack.config.main.prod.ts",
     "build:renderer": "cross-env NODE_ENV=production TS_NODE_TRANSPILE_ONLY=true webpack --config ./webpack_configs/webpack.config.renderer.prod.ts",
-    "install-python-deps": "pip3 install setuptools wheel",
+    "install-python-deps": "uv pip install setuptools wheel",
```

Comment:
let's add `uv env &&` before uv pip, for first time.

Reply:
Sure, can add `uv venv + source ...`